### PR TITLE
update Audio+ for new dashboard

### PR DESCRIPTION
**Pull request by AprilSylph**
brings back the volume control slider, and hides the pop-out audio option since pop-out audio is a real feature now

bonus: the volume slider now gets added on peepr and on the audio pop-out!

### CODE REVIEW

File: Extensions/audio_plus.js [hunk 1]
```diff
@@ -365,18 +412,26 @@ XKit.extensions.audio_plus = {
...
-		XKit.post_listener.remove("audio_plus");
-		window.removeEventListener("scroll", this.handle_scroll, false);
+		if (this.pop_out_controls) {
+			XKit.tools.add_function(function() {
+				clearInterval(window.xkit_audio_plus_check_current_interval);
+			}, true, "");
+			$("#xkit-audio-plus-current-player").remove();
+
+			document.body.removeChild(this.pop_out_controls);
+			window.removeEventListener("scroll", this.handle_scroll, false);
```
Comment (hobinjk) [inline Extensions/audio_plus.js:430]: Looks like this section is missing a removal of this listener:
```javascript
 			$(document).on("mousemove mousedown mouseup mouseout click", ".xkit-audio-plus-slider", this.slider_handle_event);
```

**Comment by AprilSylph** [comment]
weird, i specified that it did work on peepr now, so i assume it did when i PR'd this...

**Comment by hobinjk** [comment]
Yeah, the bonus appears to have disappeared in the mists of Tumblr. It might be a Firefox/Chrome difference since my development setup currently uses the bad one

**Comment by AprilSylph** [comment]
looks like this is a browser difference, still works on peepr and popout for me. tumblr seems to be eating the `audio_plus_done` class when doing popout stuff though so i'll at least need to fix that repetition prevention

**Comment by AprilSylph** [comment]
not sure if it's my last commit or not, but i'm seeing this work flawlessly on both firefox 68.9.0esr and chromium 83.0.4103.106 for both peepr and pop-out audio

**Pull request merged**